spark/src/main/scala/com/alibaba/graphar/utils/IndexGenerator.scala

```diff
@@ -27,11 +27,17 @@ import org.apache.spark.rdd.RDD
 import scala.collection.SortedMap
 import scala.collection.mutable.ArrayBuffer
 
+/** IndexGenerator is a object to help to generate the indices for vertex/edge DataFrames. */
 object IndexGenerator {
 
   // index helper for the vertex DataFrame
 
-  // return a DataFrame contains two columns: vertex index & primary key
+  /** Generate a vertex index mapping of primary key.a DataFrame contains two columns: vertex index & primary key
```

Comment:
Generate a vertex index mapping from the primary key, the result DataFrame contains two columns: vertex index & primary key